src/main/java/org/terasology/ligthandshadow/componentsystem/controllers/AttackSystem.java

```diff
@@ -146,6 +103,17 @@ private boolean itemIsFlag(EntityRef checkedItem) {
         return (checkedItem.hasComponent(BlackFlagComponent.class) || checkedItem.hasComponent(RedFlagComponent.class));
     }
 
+    private String checkWhichFlagPicked(InventorySlotChangedEvent event) {
+        item = event.getNewItem();
+        if (item.hasComponent(BlackFlagComponent.class)) {
+            return LASUtils.BLACK_TEAM;
+        }
+        if (item.hasComponent(RedFlagComponent.class)) {
+            return LASUtils.RED_TEAM;
+        }
+        return null;
+    }
```

Comment:
I wonder whether we might want to drop the `BlackFlag` and `RedFlag` components and rather have a component class member `team` or so, that holds the respective URI we return here...